bp_common/syn/Makefile.regress

```diff
@@ -57,6 +109,7 @@ RV64_BENCHMARKS := \
                    vvadd
 
 RV64_ISA_TEST_ROMS   := $(addsuffix _rom, $(RV64_ISA_TESTS))
+RV64_ISA_V_TEST_ROMS := $(addsuffix _rom, $(RV64_V_ISA_TESTS))
 RV64_BENCHMARK_ROMS  := $(addsuffix _rom, $(RV64_BENCHMARKS))
-RV64_REGRESSION_ROMS := $(RV64_ISA_TEST_ROMS) $(RV64_BENCHMARK_ROMS) 
+RV64_REGRESSION_ROMS := $(RV64_ISA_TEST_ROMS) $(RV64_ISA_V_TEST_ROMS) 
```

Comment:
We still need the benchmarks in regression

Reply:
I removed this for now because I wasn't testing the benchmarks yet, will revert.